Contact and stability sharp ib (#339)
lethe-cfd/lethe: doc/source/parameters/resolved_cfd-dem/resolved_cfd-dem.rst

* The ``alpha`` parameter is the relaxation parameter used when solving the dynamics equation of the particle.

* The ``fluid density`` parameter is the fluid density used in the force calculation of the resolved CFD-DEM. This parameter is redundant with others in the solver and will be removed in the upcoming code modification.

* The subsection ``gravity`` defines the value of the gravity used in the simulation. This gravity can be defined as a function that evolves in time and space. Each component of the ``Function expression`` corresponds respectively to its magnitude in X, Y, and Z.

The following properties are used if the particle impact one of the boundaries of the domain. The effective properties used for calculating the impact force are calculated using a harmonic mean of the properties of the wall and the particle.

* The ``wall friction coefficient`` parameter is the coefficient of friction of the wall. This parameter is used to define the effective coefficient of friction between the wall and the particles. At This point in time, all the walls have the same properties.

> * The ``wall poisson ratio`` parameter is the Poisson ratio of the wall's material. This parameter is used to define the nonlinear spring constant used when a particle impacts a wall. At This point in time, all the walls have the same properties.

Reviewer: ```suggestion
* The ``wall poisson ratio`` parameter is the Poisson's ratio of the wall's material. This parameter is used to define the nonlinear spring constant used when a particle impacts a wall. At This point in time, all the walls have the same properties.
```

* The ``wall restitution coefficient`` parameter is the restitution coefficient of the wall's material. This parameter is used to define the effective restitution coefficient for the impact of a particle and the wall. At This point in time, all the walls have the same properties.

> * The ``wall rolling friction coefficient`` parameter is the sliding friction coefficient of the wall. This parameter is used to define the effection friction coefficient between the wall and the particles when their surfaces are sliding relative to each other. At This point in time, all the walls have the same properties.

Reviewer: ```suggestion
* The ``wall rolling friction coefficient`` parameter is the rolling friction coefficient of the wall. This parameter is used to define the effective rolling friction coefficient between the wall and the particles. At This point in time, all the walls have the same properties.
```

> * The ``wall youngs modulus`` parameter is the young modulus of the wall's material. This parameter is used to define the nonlinear spring constant used when a particle impacts a wall. At This point in time, all the walls have the same properties.

Reviewer: ```suggestion
* The ``wall youngs modulus`` parameter is the Young's modulus of the wall's material. This parameter is used to define the nonlinear spring constant used when a particle impacts a wall. At This point in time, all the walls have the same properties.
```

* The ``radius`` parameter is used to define the radius of this particle.

The following properties are used if the particle impact one of the boundaries of the domain or another particle. The effective properties used to calculate the impact force are calculated using a harmonic mean of the properties of the particle and the object it impacts.

* The ``friction coefficient`` parameter is the coefficient of friction of the particle. This parameter is used to define the effective coefficient of friction between the wall and the particles.

> * The ``poisson ratio`` parameter is the Poisson ratio of the particle's material. This parameter is used to define the nonlinear spring constant used when a particle impacts a wall.

Reviewer: ```suggestion
* The ``poisson ratio`` parameter is the Poisson's ratio of the particle's material. This parameter is used to define the nonlinear spring constant used when a particle impacts a wall.
```

* The ``restitution coefficient`` parameter is the restitution coefficient of the particles' material. This parameter is used to define the effective restitution coefficient for the impact of a particle and the wall.

> * The ``rolling friction coefficient`` parameter is the sliding friction coefficient of the particle. This parameter is used to define the effection friction coefficient between the wall and the particles when their surfaces are sliding relative to each other. The effective coefficient is calculated using a harmonic mean of the properties of the particles and the other objects it impacts.

Reviewer: ```suggestion
* The ``rolling friction coefficient`` parameter is the rolling friction coefficient of the particle. This parameter is used to define the effective rolling friction coefficient between the wall and the particles. The effective coefficient is calculated using a harmonic mean of the properties of the particles and the other objects it impacts.
```

> * The ``youngs modulus`` parameter is the young modulus of the particle's material. This parameter is used to define the nonlinear spring constant used when a particle impacts a wall.

Reviewer: ```suggestion
* The ``youngs modulus`` parameter is the Young's modulus of the particle's material. This parameter is used to define the nonlinear spring constant used when a particle impacts a wall.
```
